community/governance/index.md

```diff
@@ -5,4 +5,125 @@ sidebar:
   nav: community
 ---
 
-* [SIGs](/community/governance/sigs)
+{% include toc %}
+
+The Spinnaker OSS project is run by a group of appointed members who hold specific roles, managed by the technical oversight committee. The steering committee manages the governance structure of the project, revisiting and refining it as needs evolve.
+
+The broader community is organized around a set of special interest groups (SIGs) focused on particular areas of interest.
```

Comment:
"...is organized *into* a set of" sigs?

Link from "special interest groups (SIGs)" to the SIGs section below?